web/craco.config.js

```diff
@@ -0,0 +1,20 @@
+const CracoLessPlugin = require('craco-less');
+
+module.exports = {
```

Comment:
Using [CRACO](https://github.com/gsoft-inc/craco) to overwrite the less variables